packages/gasket-plugin-happyfeet/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "gasket-plugin-happyfeet",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "test": "cross-env NODE_OPTIONS='--unhandled-rejections=strict' jest"
+  },
+  "keywords": [],
+  "author": "",
```

Comment:
```suggestion
  "author": "GoDaddy Operating Company, LLC",
```

Reply:
Also, feel free to add your name to a `maintainers` property, [like this](https://github.com/godaddy/gasket/blob/main/packages/gasket-plugin-morgan/package.json#L32-L34).